distribution/docs/src/main/jdocs/content/_plugins/geocoder-plugin-contents.adoc

```diff
@@ -0,0 +1,33 @@
+:type: plugin
+:status: published
+:title: GeoCoder Plugin
+:link: _geocoder_plugin
+:plugintypes: preingest
+:summary: Populates the `Location.COUNTRY_CODE` attribute if the Metacard has an associated location.
+
+The GeoCoder Plugin is a pre-ingest plugin that is responsible for populating the Metacard's `Location.COUNTRY_CODE` attribute if the Metacard has an associated location.
```

Comment:
Yes. I think it should be lowercase except in section headings and in plugin names. I'll do a find/replace.